.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,39 @@
+name: CIFuzz
+
+on:
+  pull_request:
+    branches: [master]
+
+permissions:
+  contents: read
+
+jobs:
+  Fuzzing:
+    runs-on: ubuntu-latest
+    if: github.repository == 'secdev/scapy'
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.ref }}
+      cancel-in-progress: true
+
+    steps:
+      - name: Build Fuzzers
+        id: build
+        uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
```

Comment:
Looking at https://github.com/secdev/scapy/actions/workflows/cifuzz.yml it seems it isn't allowed. What's weird is that the first attempt was successful.

FWIW I think it should already be useful.

(I also pushed a commit turning `print` to `log_runtime.warning`. It's a minor thing but it should help to make the fuzz target less chatty. It's just something I noticed looking at the logs)

Reply:
If you ca  re trigger it, should be okay by now.
I've indeed enabled the more 'secure' options of Github Actions on secdev/scapy.

Reply:
Got it.

Looks like `google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master` is blocked.

Reply:
Should now be allowed.

Reply:
Looks like it works. It also caught https://github.com/secdev/scapy/pull/4396#discussion_r1611498971 so I think it's more or less ready.